ui/src/components/common/DonutChart.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { PieChart } from 'react-minimal-pie-chart';
+
+export type IDonutChartData = {
+    title: string,
+    value: number,
+    color: string
+}
+
+export const donutChartColors = ['#835AF7', '#5AF7AC', '#FF29D0', '#131720', '#CBD0CB']
+
+const DonutChart = (props: {
+    data: IDonutChartData[]
+}) => (
+    <PieChart
+        radius={30}
+        animate
+        lineWidth={25}
+        data={props.data}
+        labelPosition={110}
+        viewBoxSize={[100,100]}
+        label={({ dataEntry }) => `${dataEntry.title} ${dataEntry.value.toFixed(2)}`}
+    />
```

Comment:
consider wrapping this in a div 

```
<div className='donut-chart'>
   <PieChart .../>
</div>
```

so that the mixin in the scss file can have a single `.donut-chart { ... }` root